tests/Common/ComponentsTest.php

```diff
@@ -38,10 +38,39 @@ public function setUp()
         }
     }
 
-    public function testGetComponentDetail()
+    public function componentsProvider()
     {
+        yield 'defaultBranch' => [
+            function (\Keboola\StorageApi\Client $client) {
```

Comment:
Aby to nemusela byt anonymni funkce, ktere se predava argument `$this->_client`, tak by opet bylo potreba zmenit `$this->_client` na **public**. Je to OK?

Reply:
Tak ne, `$client` se tam musi dostat az z testu, protoze `dataProvider` se spousti z jine classy a jeste pred `setUp` metodou, takze `$this->_client` je `null`.